docs/README.md

```diff
@@ -0,0 +1,110 @@
+
+**LongITools `methyl` uses the following dictionary kind which are NOT directly supported by dsUpload, see [dsUploadMethyl](https://github.com/lifecycle-project/ds-upload-methyl)**
+
+- `cordblood`
+- `peripheral-blood`
+- `placenta`
+
+The dictionary naming of versions is as follows: **dictionary-major_dictionary-minor_dictionary-kind.xlsx**
+
+For example:
+
+- `1_0_non_rep.xslx`
+- `1_0_weekly_rep.xslx`
+- `1_0_monthly_rep.xslx`
+- `1_0_trimester_rep.xslx`
+- `1_0_yearly_rep.xslx`
+
+Semantic versioning is used with the following implementation: *(Note: is this correct? row instead of column)*
+
+- **dictionary-major**
+  - Remove columns from the existing tables
+- **dictionary-minor**
+  - Add additional columns to the existing tables
+  - Renaming of columns withing the existing tables (the original column remains in the set)
+  - Changing the column type within the existing tables
+
+## Dictionary excel file
+
+The dictionary should be provided as an excel file (.xslx) with two sheets:
+
+- Variables
+- Categories
+
+## Dictionary columns
+
+The following columns are allowed:
+
+- Variables (sheet)
+  - name (`variable_name`, `repeat_name_0`, `cats_`)
+  - valueType (`decimal`, `integer`, `text`)
+  - unit ([see list](https://data-catalogue.molgeniscloud.org/CatalogueOntologies/tables/#/Units))
+  - label (eg. [Cat ownership in child's household from 0 to <1 year](https://data-catalogue.molgeniscloud.org/catalogue/catalogue/#/networks-catalogue/EUChildNetwork/variables/cats_0?model=LifeCycle_CDM&version=1.0.0))
+
+- Categories (sheet)
+  - variable (`cats_`)
+  - name (`0`)
+  - isMissing (`=FALSE()`)
+  - label (`no`)
+
+## Dictionary template
+
+*NOTE: add empty template*
+
+## Variables
+
+Non repeating and repeating variables follow a slightly different format. A non repeating variable will look like `variable_name`, lower case and connected by an underscore.
+
+Repeating variables are defined like `repeat_name_`, lower case and postfix with an underscore. dsUpload will accept variables like `repeat_name_0`, `repeat_name_1`.
+
+Some variables are reserved by dsUpload:
+
+- `row_id`
+- `child_id`
+- `age_years`
+- `age_months`
+- `age_weeks`
+- `age_trimester`
+
+## Changelogs
+
+To keep track of all the changed within the different versions of the dictionaries and data releases need to have changelogs. This way we can trace back what has happened in which release.
+
+Changelogs are placed in the [directory](https://github.com/lifecycle-project/ds-dictionaries/tree/master/changelogs) `changelogs` and should contain at minimum the following:
+
+- Explanation where the variables came from and which network/project defined these variables.
+- Version of the published dictionary
+- Which version of dsUpload is compatible with the dictionary
+- List of General changes
+- List of Additional variables which in case of a new dictionary should list ALL variables for each dictionary kind, list Variable, Type and Description.
```

Comment:
List of additional variables. In case of a new dictionary this list should contain ALL variables for each dictionary, (including name, type and description).